Tests/UIKitTests/UIViewControllerExtensionsTests.swift

```diff
@@ -54,6 +54,18 @@ final class UIViewControllerExtensionsTests: XCTestCase {
         NotificationCenter.default.post(name: notificationIdentifier, object: nil)
         XCTAssertFalse(viewController.notificationFired)
     }
+    
+    class MyViewController: UIViewController {
+        var foo: String?
+    }
+    
+    func testInstantiate() {
+        let bar = "Bar"
+        let vc = MyViewController.instantiate(from: "TestStoryboard")
+        XCTAssertNotNil(vc)
+        vc.foo = bar
+        XCTAssertEqual(vc.foo, bar)
```

Comment:
It feels like you're doing this to test that you have a `MyViewController` object. Just check
```Swift
XCTAssert(vc is MyViewController)
```
You can also then remove the `foo` property from the class, as it's not really necessary.

Reply:
But based on the method signature, this will be always a `MyViewController` so not sure this assertion would do something ... Maybe we could do two tests here one like this asserting no throw and one where we try to instantiate a view controller that isn't on the storyboard so we expect a force unwrap error or an assertion. What you think @guykogus ?